picard/util/__init__.py

```diff
@@ -1321,7 +1321,7 @@ def parse_versioning_scheme(versioning_scheme):
     from picard import log
 
     if versioning_scheme == 'semver':
-        pattern = r'^\D*\d+\.\d+\.\d+$'
+        pattern = r'^\D*\d+\.\d+(\.\d+)?$'
```

Comment:
We should perhaps even go further and fully support https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
Yes, we could support semver with relaxed syntax (a.b vs a.b.c), not sure Picard-like versions will be used by plugin devs (we can always add a versioning_scheme for it).
And yes, each versioning_scheme has its own sorting logic.